### PR TITLE
Adding guzzle requirement note

### PR DESCRIPTION
**Pull request by adamgoose**

### CODE REVIEW

File: artisan.md [hunk 1]
```diff
@@ -170,3 +170,7 @@ In this example, the `foo` command will be run every minute if it is not already
 #### Ping A Given URL After The Job Runs
 
 	$schedule->command('foo')->thenPing($url);
+
+Using the `thenPing($url)` feature require that the Guzzle 4 HTTP library be installed into your application. You can add Guzzle 4 to your project by adding the following line to your `composer.json` file:
```
Comment (taylorotwell) [inline artisan.md]: I'm pretty sure it's Guzzle 4 or 5, and the "require" should be "requires".

Comment (adamgoose) [inline artisan.md]: Okay, will update here *and* in the `mail.md`, because that's where I copied this from. ;)

**Comment by adamgoose** [comment]
Does that look good, or would you rather I use `4.*|5.*`?

**Comment by GrahamCampbell** [comment]
We're only recommending guzzle 5 in composer - ie, we're saying we don't support guzzle 4. I think we should document it like that too.

**Comment by GrahamCampbell** [comment]
Even though it would probably work with guzzle 4.

**Comment by adamgoose** [comment]
Okay, so if @taylorotwell agrees, I will remove the recommendation for Guzzle 4, and update `mail.md` to reflect as well.

**Comment by taylorotwell** [comment]
Yes, agree.

**Comment by taylorotwell** [comment]
The "require" typo still wasn't fixed.

**Comment by adamgoose** [comment]
Bahaha sorry... Sending another now.

**Comment by taylorotwell** [comment]
I went back and fixed it. Just don't merge things until I can sign off on them.

**Pull request merged**